src/containers/navigation-icons/user-icons/UserIcons.tsx

```diff
@@ -21,16 +23,25 @@ interface UserIconsProps {
 const UserIcons: FC<UserIconsProps> = ({ setSidebarOpen }) => {
   const [notificationItems, setNotificationItems] =
     useState<Notification[]>(mockNotifications)
-  const [menuAnchorEl, setMenuAnchorEl] = useState<HTMLElement | null>(null)
-  const [notificationsAnchor, setNotificationsAnchor] =
-    useState<HTMLElement | null>(null)
-  const anchorRef = useRef<HTMLDivElement | null>(null)
+  const {
+    anchorEl: accountenuAnchorEl,
```

Comment:
```suggestion
    anchorEl: accountMenuAnchorEl,
```
I think it's a typo here

Reply:
Your right. Fixed